### ISSUE TITLE
Batch Request Throttling Behaviour

### ISSUE THREAD

**Issue by verhyppo** [microsoftgraph/microsoft-graph-docs#4694]
This page does not cover the requests batching behaviour. 

As far as I can understand, because one batch request is multiplexed to multiple requests, the behaviour is similar to the one explained in the page, but:

* How long should we wait when a batch request is throttled? (e.g. the sum of all Retry-After reply headers? Double of the maximum Retry-After response headers?)
* The batch request itself, that always returns 200, affects the throttling behaviour?
* How should we manage batch request that continue to fail indefinitely? 

Thanks.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 8df32e5f-8ccb-0c93-28d9-02392f9c7256
* Version Independent ID: 4aadd084-fb61-e89b-f63f-3b71622636b7
* Content: [Microsoft Graph throttling guidance - Microsoft Graph](https://docs.microsoft.com/en-us/graph/throttling)
* Content Source: [concepts/throttling.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/master/concepts/throttling.md)
* Product: **non-product-specific**
* Technology: **microsoft-graph**
* GitHub Login: @piotrci
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by svrooij** [comment]
Just a note, if you do a batch request you'll get the statuscode per item in your batch request. While the batch request may give a Status 200, it's possible to have some items respond with a 200 or 204 and some items with a 429.

**Comment by baywet** [comment]
Thank you for your feedback and patience on the matter.
Over the last few months, we've added a lot of content to the throttling page. (mostly service limits)
I've authored PR #9560 to document the behavior when batched request get throttled. When the PR gets merged, it'll close this issue. Feel free to leave some feedback on the PR itself.
Just that you know, the service is currently experiencing an issue with batches, throttling and outlook resources. More information #8760
